.travis.yml

```diff
@@ -15,6 +15,7 @@ env:
   - TEST_SUITE=spec
   - TEST_SUITE=spec:javascript
   - TEST_SUITE=spec:compile
+  - TEST_SUITE=spec:jest
```

Comment:
I was unable to find a way to have a different language (besides ruby) for travis to run while keeping the ruby one, this means we execute the tests as a rake task under ruby environment, however I believe this is exactly how jasmine tests run today.

It has a few disadvanteges:
1. setup installs the entire environment, when in fact we only need yarn (adds about 10 minutes to the test)
2. we run jest twice with two different rubies.
3. we can't run with different node version.

@skateman - you once mentioned you know of a way to run travis with different language matrix... any idea? :)

Reply:
@ohadlevy yes, look [here](https://github.com/skateman/purr/blob/master/.travis.yml) and [here](https://github.com/skateman/purr/blob/master/ci/before_install). Basically travis provides support for a `generic` container which is close to a clean debian/ubuntu base container. Here you can set up your stuff without interfering with the preinstalled environment.

Reply:
@skateman thanks - the way I read it:
1. we can do it, but it means we have to revisit/update how we use travis today.
2. not in this PR :)